clients/venice-push-job/build.gradle

```diff
@@ -3,6 +3,7 @@ plugins {
 }
 
 dependencies {
+  implementation project(':clients:da-vinci-client')
```

Comment:
Do you think we can move these util classes to a common package so we don't need `da-vinci-client` dependency here?

Reply:
Yeah...I agree this is not good looking. @nisargthakkar also pointed out this. Let me see how much effort is needed to pull them out. And if it is too big, it might need to happen in a follow up PR.

Reply:
For now, I think it would be a fair amount of work to move it and make this PR looks more complicated. I think it will happen in a follow up PR. I can do the Java class shuffling there.